mobile-app/src/screens/Setup/SelectTrackers/SelectTrackers.js

```diff
@@ -196,29 +202,81 @@ const SelectTrackers = () => {
               !toDosSelected && { opacity: 0.5 },
               toDosSelected && { borderWidth: 1.5 },
               toDosSelected && { borderColor: "#25436B" },
-              { width: height * 0.2, height: height * 0.2 },
+              { width: buttonSize, height: buttonSize },
             ]}
             onPress={() => setToDosSelected(!toDosSelected)}
           >
             <Image
               style={{
-                width: height * 0.08,
-                height: height * 0.08,
+                width: buttonImageSize,
+                height: buttonImageSize,
               }}
               source={require("../../../assets/images/to-dos512.png")}
             />
             <Text style={styles.title}>to-dos</Text>
           </TouchableOpacity>
         </View>
+        {DISPLAY_PHYSICAL_SETUP_BUTTONS ? 
+          <View style={styles.center}>
+            <TouchableOpacity 
+              style={[
+                styles.button,
+                {
+                  backgroundColor: "rgba(202,189,255,1)",
+                  borderColor: "rgba(181, 176, 202,1)",
+                  width: buttonSize,
+                  height: buttonSize,
+                },
+                !dietSelected && { opacity: 0.5 },
+                dietSelected && { borderWidth: 1.5 },
+                dietSelected && { borderColor: "#25436B" },
+              ]}
+              onPress={()=> setDietSelected(!dietSelected)}
+              >
+                <Image
+                  style={{
+                    width: buttonImageSize,
+                    height: buttonImageSize,
+                  }}
+                  source={require("../../../assets/images/diet512.png")}
+                />
+                <Text style={styles.title}>diet</Text>
+            </TouchableOpacity>
+            <TouchableOpacity 
+              style={[
+                styles.button,
+                {
+                  backgroundColor: "rgba(202,189,255,1)",
+                  borderColor: "rgba(181, 176, 202,1)",
+                  width: buttonSize,
+                  height: buttonSize,
+                },
+                !fitnessSelected && { opacity: 0.5 },
+                fitnessSelected && { borderWidth: 1.5 },
+                fitnessSelected && { borderColor: "#25436B" },
+              ]}
+              onPress={()=> setFitnessSelected(!fitnessSelected)}
+              >
+                <Image
+                  style={{
+                    width: buttonImageSize,
+                    height: buttonImageSize,
+                  }}
+                  source={require("../../../assets/images/fitness512.png")}
+                />
+                <Text style={styles.title}>fitness</Text>
+            </TouchableOpacity>
+          </View>
+        : null}
```

Comment:
Since we arent showing anyinthing in the "else". Use the && conditional render operator instead of the ternary ? operator